src/coprocessor/dag/expr/builtin_time.rs

```diff
@@ -91,4 +110,21 @@ mod test {
             assert_eq!(got, Datum::Bytes(exp.to_string().into_bytes()));
         }
     }
+
+    #[test]
+    fn test_date() {
+        let tests = vec![
+            ("2011-11-11", "2011-11-11"),
+            ("2011-11-11 10:10:10", "2011-11-11"),
+        ];
```

Comment:
Could you also add tests for zero dates and invalid dates?

Reply:
Done